tests/Doctrine/Tests/DBAL/Platforms/SQLServer2008PlatformTest.php

```diff
@@ -19,4 +20,20 @@ public function testGeneratesTypeDeclarationForDateTimeTz()
                 array())
         );
     }
+
+    public function testGetDefaultValueDeclarationSQLForDateType()
+    {
+        $currentDateSql = $this->_platform->getCurrentDateSQL();
+	    foreach (array('date', 'date_immutable') as $type) {
+		    $field = array(
+			    'type'    => Type::getType($type),
+			    'default' => $currentDateSql,
+		    );
+		
+		    $this->assertEquals(
```

Comment:
Please use `self::assert*` instead, and it would be better to use `assertSame()` in this case